packages/create-resolve-app/bin/index.js

```diff
@@ -201,14 +205,57 @@ if (unknownOptions && unknownOptions.length) {
     }
   }
 
-  const patchPackageJson = () =>
-    replace({
-      regex: /^(\s*"name":\s*)".*$/,
-      replacement: /\1"${appName}"/,
-      paths: [`./${appName}/package.json`],
-      recursive: false,
-      silent: false
+  const getResolvePackages = () => {
+    return new Promise((resolve, reject) => {
+      request(
+        'https://www.npmjs.com/-/search?text=maintainer:reimagined&size=100',
+        { json: true },
+        (fetchError, response, body) => {
+          if (fetchError) {
+            reject('Package list loading error:' + fetchError.stack)
+          }
+          try {
+            resolve(body.objects.map(object => object.package.name))
+          } catch (parseError) {
+            reject('Package list loading error:' + parseError.stack)
+          }
+        }
+      )
     })
+  }
+
+  const patchPackageJson = () => {
+    log()
+    log(chalk.green('Patch package.json'))
+
+    let packageJsonPath = `${process.cwd()}/${appName}/package.json`,
+      packageJson = require(packageJsonPath)
+
+    packageJson.name = appName
+    packageJson.version = resolveVersion
+
+    return getResolvePackages()
+      .then(packages => {
+        Object.keys(packageJson.dependencies).forEach(k => {
+          if (packages.indexOf(k) > -1) {
+            packageJson.dependencies[k] = resolveVersion
+          }
+        })
+      })
+      .then(() =>
+        fs.writeFileSync(packageJsonPath, JSON.stringify(packageJson, null, 2))
+      )
+  }
+
+  const install = () => {
+    log()
+    log(chalk.green('Install dependencies'))
+    let command = `cd ./${appName} && npm i`
```

Comment:
Optional: Maybe use that package manager, which had been used within `create-resolve-app` invocation, depend on `process.argv` in current process?